examples/chart/teleport-agent/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+name: teleport-agent
+apiVersion: v2
+version: 0.0.1
```

Comment:
Consider also adding an `appVersion` field to the manifest to specify Teleport version itself? This way a user will be able to easily see Teleport version when running `helm ls` for example. The downside I guess is that it will have to be updated whenever we bump the image this chart uses.

Reply:
We've recently started using the generic 4.3/4.4/5.0 tagged images for the charts (which are automatically rebuilt every night with the latest version of Teleport in that branch) so this isn't a huge problem. I like the idea in general and might adopt it for the other charts.

Reply:
Added `appVersion` here and also used it as the image tag in the deployment.
There's still a `teleportVersionOverride` in `values.yaml` if it's needed.

I hope this won't become a maintenance burden - yet another field to update when we make a release.

Reply:
It will, but we can script it.